Solaris specific: modify default font to Courier to match mp(1) output

diff --git a/src/paps.1 b/src/paps.1
index b804db3..b997556 100644
--- a/src/paps.1
+++ b/src/paps.1
@@ -50,7 +50,7 @@ Format output into \fIcl\fR columns. Default is 1.
 Please notice this option isn't related to the terminal length as in a "80 columns terminal".
 .TP
 .B \-\-font=desc
-Use font based on description \fIdesc\fR. Default is \fB"Monospace 12"\fR.
+Use font based on description \fIdesc\fR. Default is \fB"Courier 10"\fR.
 The format of the string representation is:
 .IP
 "[family-list] [style-options] [size]"
diff --git a/src/paps.c b/src/paps.c
index d7315d7..4e28d45 100644
--- a/src/paps.c
+++ b/src/paps.c
@@ -57,10 +57,15 @@ int wcwidth(wchar_t c);
 #endif
 
 #define BUFSIZE 1024
-#define DEFAULT_FONT_FAMILY     "Monospace"
-#define DEFAULT_FONT_SIZE       "12"
-#define HEADER_FONT_FAMILY      "Monospace Bold"
-#define HEADER_FONT_SCALE       "12"
+#define DEFAULT_FONT_FAMILY     "Courier"
+/*
+ * Size 10 is a value matching the output of mp(1).
+ * With PANGO_SCALE of 1024, this is 7680.
+ * By PANGO_PIXELS(), this is 8px.
+ */
+#define DEFAULT_FONT_SIZE       "10"
+#define HEADER_FONT_FAMILY      "Courier Bold"
+#define HEADER_FONT_SCALE       "10"
 #define MAKE_FONT_NAME(f,s)     f " " s
 
 /*
@@ -551,7 +556,7 @@ int main(int argc, char *argv[])
     {"columns", 0, 0, G_OPTION_ARG_INT, &num_columns,
      N_("Number of columns output. (Default: 1)"), "NUM"},
     {"font", 0, 0, G_OPTION_ARG_STRING, &font,
-     N_("Set font. (Default: Monospace 12)"), "DESC"},
+     N_("Set font. (Default: Courier 10)"), "DESC"},
     {"output", 'o', 0, G_OPTION_ARG_STRING, &output,
      N_("Output file. (Default: stdout)"), "DESC"},
     {"rtl", 0, 0, G_OPTION_ARG_NONE, &do_rtl,
